src/openrct2/core/Path.cpp

```diff
@@ -23,86 +23,81 @@
 
 namespace Path
 {
-    utf8* Append(utf8* buffer, size_t bufferSize, const utf8* src)
-    {
-        return safe_strcat_path(buffer, src, bufferSize);
-    }
-
-    std::string Combine(std::string_view a, std::string_view b)
+    u8string Combine(u8string_view a, u8string_view b)
     {
         if (a.empty())
-            return std::string(b);
+            return u8string(b);
         if (b.empty())
-            return std::string(a);
+            return u8string(a);
         auto aEnd = a.back();
         auto bBegin = b.front();
         if (Platform::IsPathSeparator(aEnd))
```

Comment:
Future refactor to replace this function with  `return (fs::path(u8path(a)) / fs::path(u8path(b))).u8string()`

Reply:
Hmm just reading all the in's and out's of fs::path /operator and it might not be as simple as that. Will need to  check all calling sites to ensure above change makes sense.